docs/the-basics-console-mode.md

```diff
@@ -5,46 +5,46 @@ title: Console Mode
 
 ## Introduction
 
-Console Mode is a productivity feature shipped with Bottender. It speeds up bot development iteration by running bots in the console. Comparing with testing your bots on existing messaging platform (e.g. Messenger, LINE), it is recommended to build, test, and debug your bot in the console mode.
+The console mode is a productivity feature shipped with Bottender. It speeds up bot development iteration by running bots in the console. Comparing with testing your bots fully on the existing messaging platforms (e.g., Messenger, LINE), we recommend building, testing, and debugging part of your bots in the console mode.
 
-## Running in Console Mode
+## Running in the Console Mode
 
-In Bottender apps that created by Create Bottender App, you can pass `--console` option from npm scripts to underlying start/dev commands:
+In Bottender applications that created by Create Bottender App, you can provide the `--console` option from npm scripts to the underlying start/dev commands:
 
 ```sh
 npm start -- --console
 npm run dev -- --console
 ```
 
-With this option, the bot will be started and run in the console:
+With the `--console` option, Bottender runs the bot in the console:
 
 ![](https://user-images.githubusercontent.com/3382565/67745487-57991c80-fa5f-11e9-8eb7-9e4144df9e73.png)
 
-## Sending Text Message and Payload
+## Sending Text Messages and Payloads
 
-The most common event types that sent by the end users are text message and payload event.
+The most common event types that sent by the end users are text message event type and payload event type.
 
-To send a text message event, enter your text into the console:
+To send a text message event, enter your message text into the console:
 
 ```
 You > MY_TEXT
 ```
 
-To send a payload event, enter your payload with `/payload` prefix:
+To send a payload event, enter your payload with the `/payload` prefix:
 
 ```
 You > /payload MY_PAYLOAD
 ```
 
-## Receiving Response
+## Receiving the Responses
 
-Every responses you receive from the bot are following by `Bot >` prefix, for example:
+Every response you receive from the bot is following by the `Bot >` prefix, for example:
 
 ```
 Bot > Hi!
 ```
 
-Except text messages, other response types will be displayed as method name and its arguments:
+Except for text message responses, Bottender displays other response types as the name and arguments of the invoked method:
```

Comment:
```suggestion
All other response types will be displayed by Bottender with the method name invoked and the arguments the method invoked with:
```

Reply:
`Bottender displays response types as the name and arguments of the invoked method, except text message responses:`